core/Plugin/Manager.php

```diff
@@ -1080,6 +1088,13 @@ private function installPluginIfNecessary(Plugin $plugin)
             $updater = new Updater();
             $updater->markComponentSuccessfullyUpdated($plugin->getPluginName(), $plugin->getVersion());
             $saveConfig = true;
+
+            /**
+             * Event triggered after a new plugin has been installed.
+             *
+             * @param string $pluginName The plugin that has been installed.
+             */
+            Piwik::postEvent('PluginManager.pluginInstalled', array($plugin));
```

Comment:
I'll change that. plugin name should be fine. Will also mention that it could be triggered multiple times.
The other events for de-/activating plugins don't have use case comments. Aren't those events self explaining?